app/views/va_facilities/_facility_status_definitions_modal.html.erb

```diff
@@ -1,6 +1,3 @@
-<a href="#facility-status-modal" aria-controls="facility-status-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
```

Comment:
this needs to be moved to wherever this is going to be used otherwise we won't have a way to open the modal or see the text (if this makes sense)

Reply:
This was refactored like the other <div tags so the modal code is only in this file now.